contracts/AllDay.cdc

```diff
@@ -483,7 +482,7 @@ pub contract AllDay: NonFungibleToken {
                 AllDay.setByID.containsKey(setID): "setID does not exist"
                 AllDay.playByID.containsKey(playID): "playID does not exist"
                 SeriesData(id: seriesID).active == true: "cannot create an Edition with a closed Series"
-                SetData(id: setID).setPlayExistsInEdition(playID: playID) != true: "set play combination already exists in an edition"
+                AllDay.getPlayTierExistsInEdition(setID, playID, tier) == false: "set play tier combination already exists in an edition"
```

Comment:
The set + play + tier duplicate check is now done directly within the smart contract.